source/_integrations/eway.markdown

```
@@ -0,0 +1,170 @@
+  device_sn: "your_device_serial"
+  device_model: "your_device_model"
+  scan_interval: 30
+```
+
+## MQTT Topics
+The integration uses the following MQTT topic structure:
+
+- {device_id}/{device_sn}/data/post - Device data updates
+- {device_id}/{device_sn}/info/post - Device information updates
+- {device_id}/{device_sn}/data/request - Request device data
+- {device_id}/{device_sn}/info/request - Request device information
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)